config/config.go

```diff
@@ -299,9 +299,8 @@ type P2PConfig struct {
 	// Time to wait before flushing messages out on the connection, in ms
 	FlushThrottleTimeout int `mapstructure:"flush_throttle_timeout"`
 
-	// Maximum size of a message packet, in bytes
-	// Includes a header, which is ~13 bytes
-	MaxPacketMsgSize int `mapstructure:"max_packet_msg_size"`
+	// Maximum size of a message packet payload, in bytes
+	MaxPacketMsgPayloadSize int `mapstructure:"max_packet_msg_payload_size"`
```

Comment:
We just changed this. Changing it back ?

Reply:
yes. my bad

Jae's original code:

| -------- 1024 bytes ----------- | amino encoding 15 bytes |
limit: 1024 + 15
a) if amino changes, you need to change the `defaultAminoOverhead` const
b) 2 TM versions with different versions of amino become incompatible

My first version

| -------- 1024 bytes ----------- | (includes amino encoded header)
limit: 1024 + 15

although this version gives you control over the size of the whole package (not just payload), if you make a mistake in the calculation (like I did with emptyPacketMsgSize, it results in "maxSize is ..., but amino unmarshal 3+...." error.

My second version

| -------- 1024 bytes ----------- | amino encoding 15 bytes |
note: limit is set to 1024+15+10. 10 bytes protect us from errors above ^^ and give the opportunity to change something in amino (make it more compact / less compact)